hugo/content/blog/2025-07-11-mumble-1-5-817.md

```diff
@@ -0,0 +1,75 @@
+* Previously, in addition to normal ACL inheritence rules, the ``Write`` ACL would always be inherited from the parent channel. For consistency, this was removed. To prevent "ungovernable" channels, users which are granted ``Write`` on the root channel, will be able to ``Write`` in any channel of the server no matter what these sub-channels configure. ({{< issue 6512 >}})
+* Previously, the ``Register`` ACL was used to determine, if a client was able to see extended user information including user IP address and user-agent. This was changed to now require the ``Ban`` ACL. ({{< issue 6697 >}})
+* Due to the client being able to uncheck both context option checkboxes in the ACL editor, admins could created "dangling" ACLs which were present, but had no effect. Starting with this release,
+clients will not be able to create these "dangling" ACLs. Existing "dangling" ACLs will not be affected by this unless they are modfied. ({{< issue 6512 >}})
+
+
+## Changelog
+
+This is the full changelog for this release (changes relative to the Mumble 1.5.735 release).
+
+### Client
+
+- Changed: Client-side max image width and height for better legibility ({{< issue 6825 >}})
+- Fixed: Bug with default values for talking UI options ({{< issue 6728 >}})
+- Fixed: Enable "Paste" action, if there is an image in clipboard ({{< issue 6790 >}})
+- Fixed: Infinite loop and log spam on ALSA input device disconnection ({{< issue 6702 >}})
+- Fixed: Limit size of images pasted in RichTextEditor ({{< issue 6790 >}})
+- Fixed: Memory leaks due to BIO_NOCLOSE ({{< issue 6604 >}})
+- Fixed: Prevent local muted users from triggering attenuation ({{< issue 6747 >}})
+- Fixed: Prevent unchecking both ACL context checkboxes ({{< issue 6512 >}})
+- Fixed: Rare crash when using PipeWire ({{< issue 6852 >}})
+- Fixed: Remove "Write" ACL parent channel inheritance ({{< issue 6512 >}})
+- Fixed: Restart audio system when a Windows audio device we use reappears ({{< issue 6777 >}})
+- Fixed: Shortcut data not being saved ({{< issue 6832 >}})
+- Fixed: Specify version when loading/saving MainWindow state ({{< issue 6584 >}})
+- Fixed: Update chat bar, if selected channel changes name ({{< issue 6748 >}})
+- Fixed: Unlink active plugin before unloading ({{< issue 6602 >}})
+- Fixed: Volume adjustment slider keyboard inputs ({{< issue 6841 >}})
+
+
+### Server
+
+- Changed: Extended user stats now requires "Ban" ACL ({{< issue 6700 >}})
+- Fixed: ACL parsing of "Traverse" and "Write" ACL ({{< issue 6512 >}})
+- Fixed: Prevent server startup when providing empty certificate files
+- Fixed: Remove "Write" ACL parent channel inheritance ({{< issue 6512 >}})
+
+
+### Miscellaneous
+
+- Added: Bundle C++ redistributables on Windows ({{< issue 6789 >}}) Many thanks to [**@sqwishy**](https://github.com/sqwishy)!
```

Comment:
Tbh, it feels a bit weird to give a shout-out to a single contributor while other contributions remain anonymous. 👀